composer.json

```
@@ -27,7 +27,7 @@
         "composer-runtime-api": "^2.0.0",
         "illuminate/cache": "^10.36|^11",
         "illuminate/container": "^10.0|^11",
-        "illuminate/database": "^10.30|^11",
+        "illuminate/database": "^10.30|>=11.0.0,<=11.16.0",
```

Waiting for https://github.com/laravel/framework/pull/52186 to get merged.

---

Let's revert, the fix have been merged.
```suggestion
        "illuminate/database": "^10.30|^11",
```